examples/Extensions/AddSitelinks.php

```diff
@@ -0,0 +1,339 @@
+        $campaignExtensionSettingOperation->setCreate($campaignExtensionSetting);
+
+        // Issues a mutate request to add the campaign extension setting.
+        $campaignExtensionSettingServiceClient =
+            $googleAdsClient->getCampaignExtensionSettingServiceClient();
+        $response = $campaignExtensionSettingServiceClient->mutateCampaignExtensionSettings(
+            $customerId,
+            [$campaignExtensionSettingOperation]
+        );
+
+        // Prints the resource name of the created campaign extension setting.
+        /** @var CampaignExtensionSetting $addedCampaignExtensionSetting */
+        $addedCampaignExtensionSetting = $response->getResults()[0];
+        printf(
+            "Created a campaign extension setting with resource name: '%s'%s",
+            $addedCampaignExtensionSetting->getResourceName(),
+            PHP_EOL
+        );
+    }
+
+    /**
+     * Creates a list of extension feed items.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param string $campaignResourceName the resource name of the campaign to target
+     * @return StringValue[] the list of extension feed items' resource names
+     */
+    private static function createExtensionFeedItems(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        // Creates a site link feed item 1.
+        $sitelinkFeedItem1 =
+            self::createSitelinkFeedItem('Store Hours', 'http://www.example.com/storehours');
+
+        // Creates an extension feed item from the sitelink feed item.
+        $extensionFeedItem1 = new ExtensionFeedItem([
+            'extension_type' => ExtensionType::SITELINK,
+            'sitelink_feed_item' => $sitelinkFeedItem1,
+            'targeted_campaign' => new StringValue(['value' => $campaignResourceName])
+        ]);
+
+        // Creates a site link feed item 2.
+        $sitelinkFeedItem2 = self::createSitelinkFeedItem(
+            'Thanksgiving Specials',
+            'http://www.example.com/thanksgiving'
+        );
+
+        // Start date is set to November 20th. If it's already passed that date for this year,
+        // use the same date for the next year instead.
+        $targetYearString = date('Y');
+        if (strtotime('now') > strtotime('20 November')) {
+            $targetYearString = date('Y', strtotime('+1 year'));
+        }
+        $startTimeString = date('Y-m-d H:i:s', mktime(0, 0, 0, 11, 20, intval($targetYearString)));
+        // Use the same year as start time when creating end time.
+        $endTimeString = date('Y-m-d H:i:s', mktime(23, 59, 59, 11, 27, intval($targetYearString)));
+
+        // Targets this sitelink for United States only.
+        // A list of country codes can be referenced here:
+        // https://developers.google.com/adwords/api/docs/appendix/geotargeting
```

Comment:
Not sure if done on purpose but this link is specific to AdWords API

Reply:
We don't have that page in Google Ads API yet.

Reply:
We don't have this page in Google Ads API yet.